src/sas/qtgui/Utilities/GridPanel.py

```diff
@@ -573,6 +573,7 @@ def newTableTab(self, tab_name=None, data=None):
 
     @classmethod
     def onHelp(cls):
```

Comment:
This should not be a class method if it needs to access self. What's going on?

Reply:
Rewrote and included regeneration of documentation with the latest commit.